common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -1439,6 +1439,22 @@ def test_join_team(self):
         self.teams_page.click_all_topics()
         self.verify_my_team_count(1)
 
+    def test_join_team_from_teams_page(self):
+        """
+        Scenario: User remains on a team's page after joing that team. See
+            TNL-3232.
+        Given I am enrolled in a course with a team
+        When I navigate to that team's page from the teams-for-topic page
+        And I join the team
+        Then I should remain on the page
+        """
+        self._set_team_configuration_and_membership(create_membership=False)
+        teams_page = BrowseTeamsPage(self.browser, self.course_id, self.topic)
+        teams_page.visit()
+        teams_page.view_first_team()
+        self.team_page.click_join_team_button()
+        self.assertTrue(self.team_page.is_browser_on_page())
+
```

Comment:
Please add tests for the two other bugs that were fixed by this change, if they aren't covered by this test.